src/compiler/transformers/ts.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path="../factory.ts" />
+﻿/// <reference path="../factory.ts" />

```

Comment:
I've looked at this and don't know why it shows a diff here.

* It only shows `^M` on lines that I've changed -- if I had changed line endings in the file, presumably they would show as changed in the whole file.
* When I look at the file, it appears to have CRLF line endings (on all lines) on both `master` and `map5`. I have `core.autocrlf` set to `false`. Tested on both Windows and Linux.

After merge I'll look back to this file to see if it has mixed line endings, but I doubt that.
CC @DanielRosenwasser